docs/user-guide/api-mediation/api-mediation-onboard-an-existing-rest-api-service-without-code-changes.md

```diff
@@ -187,17 +193,17 @@ The following list describes the configuration parameters:
         gateway endpoints. The _service-url_ parameter points to the target endpoint on the gateway.
 
 
-## Add and validate the definition in the API Mediation Layer
+## Add and validate the definition in the API Mediation Layer running on your machine
 
-After defining the service in YAML format, you are ready to add your service definition to the API Mediation Layer ecosystem.
+After you define the service in YAML format, you are ready to add your service definition to the API Mediation Layer ecosystem.
 
-Following steps show how to add your service to the API Mediation Layer on your local machine.
+The following procedure describes how to add your service to the API Mediation Layer on your local machine.
 
 **Follow these steps:**
 
-1.  Copy or move you YAML file to the `config/local/api-defs` directory in the directory with API Mediation layer.
+1.  Copy or move your YAML file to the `config/local/api-defs` directory in the directory with API Mediation layer.
 
-2.  Run the following services to onboard your application:
+2.  Run the following services to start the API Mediation Layer:
```

Comment:
What services?

Reply:
It should be `Start the API Mediation Layer services.` I will fix it